src/mesh/features/cert-manager.md

```diff
@@ -0,0 +1,111 @@
+using a CA root certificate and key stored in a HashiCorp Vault
+server.
+
+* [`acmpca`](/mesh/{{page.kong_version}}/features/acmpca): {{site.mesh_product_name}} generates data plane certificates
+using Amazon Certificate Manager Private CA.
+
+* [`certmanager`]: {{site.mesh_product_name}} generates dataplane certificates
+using Kubernetes [cert-manager](https://cert-manager.io) certificate controller.
+
+## cert-manager CA mode
+
+In `certmanager` mTLS mode, {{site.mesh_product_name}} communicates with a locally installed cert-manager `Issuer`,
+which generates the data plane proxy certificates automatically.
+{{site.mesh_product_name}} does not retrieve any CA private keys,
+which means that the private key of the CA is secured by cert-manager itself,
+or an upstream CA,
+and not exposed to third parties.
+
+In `certmanager` mode, you provide {{site.mesh_product_name}} with a cert-manager `Issuer`
+or `ClusterIssuer` reference. {{site.mesh_product_name}} will communicate with cert-manager
+via Kubernetes resources which reference the `Issuer` or `ClusterIssuer`.
+
+When {{site.mesh_product_name}} is running in `certmanager` mode, the backend communicates with cert-manager
+and ensures data plane certificates are issued and rotated for each proxy.
+
+`certmanager` is only available for {{site.mesh_product_name}} in the Kubernetes environment.
+
+### Configure cert-manager CA
+
+The `certmanager` mTLS backend requires a cert-manager `Issuer` or `ClusterIssuer` to be accessible
+from the {{site.mesh_product_name}} system namespace.
+A `ClusterIssuer` is accessible from the entire Kubernetes `cluster`.
+An `Issuer` is only accessible from within its own namespace,
+and must reside in the {{site.mesh_product_name}} system namespace
+o be available to {{site.mesh_product_name}}.
+
+If the cert-manager is configured outside of the {{site.mesh_product_name}} system namespace,
+and `ClusterIssuer` is not used,
+this may require cert-manager configuration and secrets to be "reflected" into the `Issuer`
+in the {{site.mesh_product_name}} system namespace. See https://cert-manager.io/docs/faq/sync-secrets/ for details.
+
+The following steps show how to configure cert-manager for {{site.mesh_product_name}} with
+a mesh named `default`. For your environment, replace `default` with the appropriate mesh name.
+
+See [cert-manager.io](https://cert-manager.io) to learn how t
+install and configure cert-manager.io.
+
+Once created, you will need the `IssuerRef` information of the `Issuer` or `ClusterIssuer`
+to configure {{site.mesh_product_name}}
+
+### Configure Mesh
+
+Here's an of mTLS configuration with `certmanager` backend
+which references an `Issuer` named `my-ca-issuer`:
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: Mesh
+metadata:
+  name: default
+spec:
+  mtls:
+    enabledBackend: certmanager-1
+    backends:
+    - name: certmanager-1
+      type: certmanager
+      dpCert:
+        rotation:
+          expiration: 5m
```

Comment:
Worth not giving examples with such extreme params (people blindly copy paste)

Reply:
Hah yeah I just caught this while proof-reading myself too. Changed to `24h`